src/bitboxbase/bitboxbase_status.c

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Shift Cryptosecurity AG
```

Comment:
also move into bitboxbase/components?

Reply:
later

Reply:
Actually I think we should distinguish between components and screens. but that is a later problem

Reply:
so far we've distinguished between components  and workflows, which are components+user interactions to perform an action.

the new components you added seem to be both in one. 

what do you mean by screen?

Reply:
I mean components you use with `ui_screen_stack_push/pop`.

Hmm maybe parts of what i have written should be workflows. I really wish the worklows were async though.